sdk/src/main/java/com/mapbox/maps/StyleObserver.kt

```diff
@@ -56,13 +56,11 @@ internal class StyleObserver(
       }
       awaitingStyleLoadListeners.clear()
```

Comment:
Should we keep this?

Reply:
yes, the onStyleLoaded listener should only be triggered once.